schema/deploy/shipping/views.sql

```diff
@@ -13,37 +13,36 @@ begin;
 -- there needs to be a lag between view development and consumers being
 -- updated, copy the view definition into v2 and make changes there.
 
+drop view shipping.reportable_condition_v1;
 create or replace view shipping.reportable_condition_v1 as
 
-    with reportable as (
-        select array_agg(lineage) as lineages
-        from warehouse.organism
-        where details @> '{"report_to_public_health":true}'
-    )
-
     select
         presence_absence.presence_absence_id,
         organism.lineage::text,
         sample.identifier as sample,
-        barcode,
```

Comment:
I think we should bump the revision number because we're removing existing columns from the view.

Reply:
Since we're the only consumers of this view I didn't think the version number matters

Reply:
We do have at least two Metabase queries that use this view.

Reply:
ohhh right...I guess we'll keep v1 and I'll update to v2

Reply:
Actually I'll just rename to `reportable_hcov19_v2` since this new view is specific to hCoV19

Reply:
> We do have at least two Metabase queries that use this view.

@kairstenfay Which queries? I didn't see these when I went searching.

Reply:
I'm sorry, I thought the linked Metabase queries in the Slack messages used the `shipping.reportable_condition_v1` view. I guess I remembered incorrectly.